core/src/main/resources/openapi-specs/eth2/signing/schemas.yaml

```diff
@@ -421,6 +422,18 @@ components:
       required:
         - version
         - block
+    BlockRequestBellatrix:
+      type: object
+      properties:
+        version:
+          type: string
+          description: 'version to identify block request type.'
+          example: BELLATRIX
```

Comment:
Can a enum be used to make this more clear there is only one value possible? Perhaps a enum on the block v2 type would be clearer though i.e. so we can document that Block V2 can be ALTAIR or BELLATRIX

Reply:
This is due to limitation of vertx openapi libraries. vertx openApi libraries doesn't process `oneOf` within another `oneOf`. Hence we trick it slightly using `anyOf` instead. However, within our code (and deserializer), we are using `oneOf` semantics.